Aptoide/aptoide-client-v8
# hotfix: Replaced appcoins wallet png

**What does this PR do?**

  Replaces appcoins wallet png in appcoins info view for a better one

**Database changed?**

    No

**Where should the reviewer start?**

AppCoinsInfoFragment.java
or
appcoins_wallet_icon.png

**How should this be manually tested?**

 Put false true in the if conditions in AppViewAppcInfoViewHolder and press the view that says get APPC, then check if the image is not pixelised in the card view

**What are the relevant tickets?**

  None

**Code Review Checklist**
